app/mixins/notifications-unread-count.js

```diff
@@ -0,0 +1,15 @@
+import {gt} from '@ember/object/computed';
+import Mixin from '@ember/object/mixin';
+import {computed} from '@ember/object';
+
+export default Mixin.create({
+	unreadCount: computed('notifications.model.unreadCount', function () {
```

Comment:
this could be a property on notifications service
has Unread too
this we we could :fire: :fire: this mixin!
plx plx :)